src/TodoListItem.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function TodoListItem(props) {
```

Comment:
Answering question about props naming:
In this case, `props` can be named anything, but it’s mostly a convention that was established before React moved to hooks. Before hooks, React used classes, where `this.props` was used to distinguish between values that were supplied as arguments to a React class component versus `this.state` stateful values created within a React class component. This convention was continued when React moved to hooks to keep it inline with classes. That said, unlike classes, there’s really nothing in hooks that is preventing you from naming it to something else.